src/server/modes/charts/plugins/datalens/preparers/flat-table/index.ts

```diff
@@ -290,7 +309,7 @@ function prepareFlatTable({
             }
 
             if (isActionParamsEnable) {
-                if (canUseFieldForFiltering(item)) {
+                if (item.canUseFieldForFiltering) {
                     if (isDateField(item)) {
```

Comment:
@kuzmadom is ok to using `isDateField` in this place?  Why not `isDateType`?

Reply:
there is not much difference what to use here

`isDateField` inside uses `isDateType`